src/main/webapp/WEB-INF/web.xml

```diff
@@ -19,6 +16,38 @@
     specific language governing permissions and limitations
     under the License.
 -->
-<web-app>
+<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee"
```

Comment:
There's a `web.xml` in `rest-server`, but it's not a duplicate. This `web.xml` is used during both classic & OSGi deployments. The REST API is overlayed, as a servlet in the same WebContext in classic and as a CXF context plugin in OSGi (both cases used as jar, not war).
The `web.xml` in `rest-server` is never used in the standard brooklyn init. It's there just for `BrooklynRestApiLauncher` which supports both servlet & war deployment modes. For the war case the jsgui is not installed at all.